### ISSUE TITLE
New wishlist "type": Anything I don't own

### ISSUE THREAD

**Issue by Tecfan** [bartervg/barter.vg#130]
# What problem does this feature address?
This feature will address the server issues that comes with having a very large wishlist (well maybe not, see below). The reason for having a very large wishlist is twofold: 1) It reflects a reality for collectors (I'm not the only one interested in this, for sure) and 2) it also gives you a higher chance of receiving offers.

## Describe a solution
A solution to this would be to have a wishlist option / option somewhere where you can tick off that you're interested in all items that you do not already own. This would be separate from wishlist entirely, so that you can still have a wishlist (but it doesn't need to contain 120k items). An image to help visualize the functionality:

![image](https://user-images.githubusercontent.com/956878/82034750-e178a200-969e-11ea-8d15-37865825ec9a.png)

Now this wouldn't be another list type feature or secondary wishlist, this is a simple button (possibly with an option to turn off non-Steam items, but that does not really matter to me). Either you show up on things you don't own, or you don't show up at all in this section. 

The matching function will register hits as if the items are in your wishlist.

**`Now does this help with server performance? I don't know, that's up to the smarter guys to figure out.`**

This was proposed by me back in 2015, and understandably you have been reluctant in adding it. Here is the original discussion (database was only 8000 items back then!): https://steamcommunity.com/groups/bartervg/discussions/0/530646715645601227/

Now that some really big wishlists exists on the platform, this is a good time to revisit this suggestion, so I am once again proposing it. If there are other solutions to this issue, feel free to comment.

## Examples of similar features
Wish list, without the list.

**Comment by Revadike** [comment]
Great idea. This will help lowering "do not want" decline reasons for my automated offers. Though, this item should be wishlisted by everyone who wants it. If only 1% uses it that wants all unowned, it has no use.

**Comment by Tecfan** [comment]
![image](https://user-images.githubusercontent.com/956878/82043048-34585680-96ab-11ea-9d79-42b29170f325.png)

It seems my wishlist has already started to stop working due to its size: These items all show up as 0 wishlisted.

**Comment by bartervg** [comment]
Isn't this covered with the unowned collection? It isn't fully implemented, but it does appear on item profiles below the wishlist section.
![image](https://user-images.githubusercontent.com/28576132/82777478-3beecc80-9e1c-11ea-919c-77579de34f4e.png)

**Comment by Revadike** [comment]
I don't think you understand the issue. Everyone has unowned collections, but not everyone want any unowned item. Some want wishlist only. There is currently no way to single out traders who want any unowned item.

**Comment by bartervg** [comment]
No, users can have wishlist only. That's what the stars are, wants wishlist only. 
![image](https://user-images.githubusercontent.com/28576132/82823806-36be6b80-9e76-11ea-8b6e-0306b034fc38.png)

Not only is it possible, it's the default setting for new users. They will have zero unowned items until opting-in. 
![image](https://user-images.githubusercontent.com/28576132/82823613-e515e100-9e75-11ea-88e5-6fa8357228fa.png).

Until the user clicks that option, they will not appear in Unowned.
![image](https://user-images.githubusercontent.com/28576132/82823972-80a75180-9e76-11ea-9040-eddb48e08ec1.png)

**Comment by Tecfan** [comment]
The difference is that you don't show up on matches, and nobody will be on a /i/game page and go through the Unowned list to send offers to these users.

**Comment by bartervg** [comment]
You do appear in matches. It looks like this 
![image](https://user-images.githubusercontent.com/28576132/82827677-7e94c100-9e7d-11ea-9bec-9c7ede7806aa.png)
Whereas when you have wishlist matches, it looks like this 
![image](https://user-images.githubusercontent.com/28576132/82827765-ad129c00-9e7d-11ea-87d4-4bc0f38248a5.png)
The different button and position, but if you have tradables that someone wants, you will appear in matches if you clicked `Games I Don't Have`. Wishlist only won't if there aren't wishlist matches.

As for nobody looking through unowned list, is that because there are hundred of users? Or is it a matter of layout?

**Comment by Tecfan** [comment]
Yes, you do show up in general /t matches, but not on specific game matches. This is why I added the entire database to my wishlist.

Here is the scenario that I have far not been able to successfully explain. This is done with my alt.

1. A random user has a random game not in Tecfan's library
2. (For this scenario I have removed the game from Tecfan's wishlist for testing purposes)
3. Tecfan has marked "Any game I don't own" on his profile and should be open to trade for this game even if it is not on the wishlist.
4. Random user goes to random game's matching page:
![image](https://user-images.githubusercontent.com/956878/82835119-63e53b00-9ec3-11ea-92ba-67c4f440cb3f.png)

5. Click "Search Matches" next to "Wishlist" OR go to your /t/m matching page and select the game:
![image](https://user-images.githubusercontent.com/956878/82835936-b45d9800-9ec5-11ea-8270-3e456399a21f.png)

You will only be shown users who has the game wishlisted (3). In some ways this makes sense since the button is next to "wishlist". But I want to appear in these matches as well, because I want all games. My workaround has been to just add everything to the wishlist.

If the "games I don't own" option allows you to be a) shown on specific game matches and b) show you on some sort of wishlist other than "unowned" then that would be perfect, but I don't think everybody using "games I don't own" want this. That is why I proposed a new "wishlist type" for collectors. I don't know exactly how it would work, but it would nice to finally just delete my wishlist...

Did any of this make any sense?

**Comment by Tecfan** [comment]
I'm not sure that all 600+ people who normally show up on "Unowned" pages will want this. Maybe the solution is to front "Unowned" more, somehow. "Include 'Unowned' matches", or option to expand game matching to unowned list?

The only thing I'm sure of is that I have gotten a lot more offers for my relatively low tier tradables since I added a gigantic wishlist, probably because I end up on people's match pages when they're trying to get rid of their 2018 Otaku bundle game.

**Comment by Revadike** [comment]
I'd like to mention this is by far the most requested/wanted/needed offer preference to reduce offer declines.
